driver-core/src/main/java/com/datastax/driver/core/Cluster.java

```diff
@@ -1385,19 +1385,57 @@ public Builder withNoCompact() {
       return this;
     }
 
+    /**
+     * Configure for Scylla Cloud Serverless cluster using configuration bundle.
+     *
+     * @param configurationFile
+     * @return
+     * @throws IOException
```

Comment:
Please add more detailed descriptions here and in places below. 

For example:
```
* @param  url  an absolute URL giving the base location of the image
* @param  name the location of the image, relative to the url argument
* @return      the image at the specified URL
```